### PR TITLE
#581 Get id of order's fields from backend to frontend

### LINKED ISSUE #581: Правь шаблон писем
В шаблоне письма, который приходит если оплачивать онлайн, нет строки комментариями. Как в обычном шаблоне. Добавь.

### PR DESCRIPTION
**Pull request by artemiy312**
Closes #581

### CODE REVIEW

File: front/js/components/order.es6 [hunk 1]
```diff
@@ -12,12 +12,7 @@
     remove: '.js-remove',
     paymentOptions: 'input[name=payment_type]',
     defaultPaymentOptions: 'input[for=id_payment_type_0]',
-    orderForm: {
-      name: '#id_name',
-      phone: '#id_phone',
-      email: '#id_email',
-      city: '#id_city',
-    },
+    orderForm: $('#order-form-full').data('fields'),
```
Comment (duker33) [inline front/js/components/order.es6]: `orderFormData` might be the more good name.
We can't do any actions with this `orderForm`. We can just get it's data

File: front/js/components/order.es6 [hunk 2]
```diff
@@ -159,14 +154,11 @@
...
-    const orderInfo = {
-      payment_type: getSelectedPayment(),
-    };
-
-    $.each(DOM.orderForm, (name, field) => {
-      orderInfo[name] = $(field).val();
-    });
-
+    const orderInfo = Object.keys(DOM.orderForm).reduce((acc, key) => {
+      acc[key] = $(DOM.orderForm[key]).val();
```
Comment (duker33) [inline front/js/components/order.es6]: it seems to me, that map is more simple alternative:
```
Object.keys(DOM.orderForm).reduce((key) => {
return key: $(DOM.orderForm[key]).val();
}
```

Maybe syntax is not fully working. It's my not good js

File: shopelectro/views/ecommerce.py [hunk 3]
```diff
@@ -18,9 +20,13 @@ class OrderPage(ec_views.OrderPage):
     cart = SECart
 
     def get_context_data(self, request, **kwargs):
+        data = super().get_context_data(request, **kwargs)
         return {
-            **super(OrderPage, self).get_context_data(request, **kwargs),
+            **data,
             'page': CustomPage.objects.get(slug='order'),
+            'order_fields': json.dumps({
```
Comment (duker33) [inline shopelectro/views/ecommerce.py]: `order_fields_html` might be the more good name.
Usually, we send to template data structures such as arrays or dicts.
But here we have plain string to render

File: front/js/components/order.es6 (hunk 2 above)
Comment (artemiy312) [reply front/js/components/order.es6]: `.map(...)` will produce array of objects `[{a: b}, {b: c}, ...]` instead of `{a: b, b:c, ...}`

**Pull request merged**